packages/react-instantsearch-hooks-dom/src/widgets/__tests__/Breadcrumb.test.tsx

```diff
@@ -0,0 +1,375 @@
+              Digital Cameras
+            </li>
+          </ul>
+        </div>
+      </div>
+    `);
+  });
+
+  test('transform the passed items', async () => {
+    const { searchClient, hierarchicalAttributes } =
+      getBreadcrumbSearchClient();
+
+    const { container } = render(
+      <InstantSearchHooksTestWrapper
+        searchClient={searchClient}
+        initialUiState={{
+          indexName: {
+            hierarchicalMenu: {
+              'hierarchicalCategories.lvl0': [
+                'Cameras & Camcorders > Digital Cameras',
+              ],
+            },
+          },
+        }}
+      >
+        <VirtualHierarchicalMenu attributes={hierarchicalAttributes} />
+        <Breadcrumb
+          attributes={hierarchicalAttributes}
+          transformItems={(items) =>
+            items.map((item) => ({ ...item, label: item.label.toUpperCase() }))
+          }
+        />
+      </InstantSearchHooksTestWrapper>
+    );
+
+    await wait(0);
+
+    expect(container).toMatchInlineSnapshot(`
+      <div>
+        <div
+          class="ais-Breadcrumb"
+        >
+          <ul
+            class="ais-Breadcrumb-list"
+          >
+            <li
+              class="ais-Breadcrumb-item"
+            >
+              <a
+                class="ais-Breadcrumb-link"
+                href="#"
+              >
+                Home
+              </a>
+            </li>
+            <li
+              class="ais-Breadcrumb-item"
+            >
+              <span
+                aria-hidden="true"
+                class="ais-Breadcrumb-separator"
+              >
+                &gt;
+              </span>
+              <a
+                class="ais-Breadcrumb-link"
+                href="#"
+              >
+                CAMERAS & CAMCORDERS
+              </a>
+            </li>
+            <li
+              class="ais-Breadcrumb-item ais-Breadcrumb-item--selected"
+            >
+              <span
+                aria-hidden="true"
+                class="ais-Breadcrumb-separator"
+              >
+                &gt;
+              </span>
+              DIGITAL CAMERAS
+            </li>
+          </ul>
+        </div>
+      </div>
+    `);
+  });
+
+  test('navigates to a parent category', async () => {
+    const { search, searchClient, hierarchicalAttributes } =
+      getBreadcrumbSearchClient();
+
+    const { container, getByText } = render(
+      <InstantSearchHooksTestWrapper
+        searchClient={searchClient}
+        initialUiState={{
+          indexName: {
+            hierarchicalMenu: {
+              'hierarchicalCategories.lvl0': [
+                'Cameras & Camcorders > Digital Cameras',
+              ],
+            },
+          },
+        }}
+      >
+        <VirtualHierarchicalMenu attributes={hierarchicalAttributes} />
+        <Breadcrumb attributes={hierarchicalAttributes} />
+      </InstantSearchHooksTestWrapper>
+    );
+
+    await wait(0);
+
+    search.mockClear();
```

Comment:
Why do you need to clear the mock?

Reply:
I did not need it originally, but now that I have a single mock for all tests I'm clearing it before each test to get an accurate number of its calls (see 773e4895).